packages/blade/src/components/Spinner/Spinner/Spinner.tsx

```diff
@@ -15,19 +26,29 @@ type SpinnerProps = {
    */
   size?: 'small' | 'medium' | 'large';
   /**
-   * Sets the aria-label for web & accessibilityLabel react-native.
+   * Sets the aria-label for web & accessibilityLabel react-native. Uses `label` as its value if it exists otherwise falls back to "Loading"
```

Comment:
```suggestion
   * Sets the aria-label for web & accessibilityLabel for react-native. Uses `label` as its value if it exists otherwise falls back to "Loading"
```